test/v1alpha1/cancel_test.go

```diff
@@ -29,7 +29,7 @@ import (
 
 	"github.com/tektoncd/pipeline/test/parse"
 
-	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
```

Comment:
looks like this test is meant for v1alpha1, maybe keep this as is?

Reply:
yeah test is meant for v1alpha1 but the status being used here were from v1beta1. Once `PipelineRunCancelled` is removed, we anyhow will have to use v1beta1 status. Moreover v1alpha1 gets converted to v1beta1 during runtime